src/lib/Zikula/Bundle/CoreBundle/Resources/config/translation.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" ?>
```

Comment:
Right, I knew you copied it from somewhere. It should be sufficient to only include the line you actually changed.